core/src/test/java/nl/nn/adapterframework/http/authentication/HttpSenderAuthenticationTest.java

```diff
@@ -467,4 +469,11 @@ public void testRetryOnResetAuthenticated() throws Exception {
 		assertEquals("200", session.getString(RESULT_STATUS_CODE_SESSIONKEY));
 		assertNotNull(result.asString());
 	}
+
+	@Test
+	@Override
+	@Disabled
+	public void testIfToStringWorks() {
+
```

Comment:
Misschien wel even een commentaartje dat deze weer weg moet zodra de classe omgetroverd is tot junit5